io_context/package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>io_context</name>
+  <version>0.0.1</version>
```

Comment:
Can you please update this to 0.0.6 to match the existing packages so new releases are made easier?